services/classifier/Dockerfile

```diff
@@ -24,33 +24,53 @@ RUN apt-get install -y cadaver
 WORKDIR /app
 COPY build/src .
 COPY build/mentors mentors
-# COPY build/checkpoint checkpoint
 COPY build/.netrc /root/.netrc
 
+# Download vector models
 RUN bash -c "mkdir -p checkpoint/vector_models && \
     cd checkpoint/vector_models && \
 	printf \"mget checkpoint/vector_models/*\nexit\" >> cmd && \
 	cadaver -r cmd ${WEBDAV_PATH} && \
 	rm cmd"
 
+# Download test files
```

Comment:
Do we need to have test files inside of the docker image? I would prefer if the test process could mount a directory for test checkpoints with the -v option

Reply:
example of a shell script (pseudocode) that mounts checkpoints and the compare script and then executes the compare script via entrypoint override

```bash
# example shell script that can run docker with mounted checkpoints
C1=$1
C2=$2

CHECKPOINT_ROOT=#$(figure out hosts checkpoint root)
COMPARE_SCRIPT=${PROJECT_ROOT}/src/test_checkpoint.py
DOCKER_IMAGE_NAME=ictuscdocker/mentorpal-classifier:latest

docker run \
        --rm \
        -it \
        -v ${CHECKPOINT_ROOT}:/app/checkpoints \
        -v ${COMPARE_SCRIPT}:/app/bin/host/test_checkpoint.py \
        --entrypoint ${COMPARE_SCRIPT} \
    ${DOCKER_IMAGE_NAME}

```